src/CrossFilter.js

```diff
@@ -81,6 +85,7 @@ export default class CrossFilter {
   initCrossFilterForAsync(dataConnector, dataTables) {
     this._dataConnector      = dataConnector
     this._cache              = new ResultCache(dataConnector) // this should gc old cache?
+    this._groupAllcache      = new ResultCache(dataConnector)
     this._dataTables         = Array.isArray(dataTables) ? dataTables : [dataTables]
```

Comment:
This will not work.  As every time you create a new groupAll instance -- you will create a new cache result.  Every time a user gets the Min/Max or creates a new count chart, they create a new instance of new cache.  It'll We could store it similarly to the dim array and pop out unused groupAlls, but that is quite a bit of work, and this solution works fine.